alertlogic/region.py

```diff
@@ -7,28 +7,52 @@
 """
 List of known regions
 """
-REGIONS = {
+DEFAULT_REGIONS = {
     "us": "https://api.cloudinsight.alertlogic.com",
     "uk": "https://api.cloudinsight.alertlogic.co.uk"
 }
 
 
 class Region():
     """
-    Abstracts an alertlogic region, for now it only represents the api endpoint url
+    Abstracts an alertlogic region, holds info on region
     """
 
-    def __init__(self, api_endpoint):
+    def __init__(self, reference, custom_region_map={}):
```

Comment:
There is no reason to manage DC mapping on the level of this class. To me, this class is responsible for providing DC settings to clients, as it was before. AFM, a client (or some factory class) has to manage possible datacenter mappings. Let's keep configuration parts out of `class Region`. Also the name of the class is confusing to me as well. Once it manages some concrete datacenter settings, it shall be called `class Datacenter` (not related to your changes).